### PR TITLE
Various enhancements

### PR DESCRIPTION
**Pull request by regislebrun**
The main contribution of this PR is a slight modifiction of OrthogonalUniVariatePolynomial::operator() leading to a 7x speed improvement.

### CODE REVIEW

File: lib/src/Base/Common/ResourceMap.cxx [hunk 1]
```diff
@@ -1243,7 +1243,7 @@ void ResourceMap::loadDefaultConfiguration()
 
   // MaximumEntropyOrderStatisticsDistribution //
   setAsBool( "MaximumEntropyOrderStatisticsDistribution-CheckMarginals", true );
-  setAsBool( "MaximumEntropyOrderStatisticsDistribution-UseApproximation", true );
+  setAsBool( "MaximumEntropyOrderStatisticsDistribution-UseApproximation", false );
```
Comment (jschueller) [inline lib/src/Base/Common/ResourceMap.cxx]: this must be changed in the config file as well

File: lib/test/t_MaximumEntropyOrderStatisticsCopula_std.cxx [hunk 2]
```diff
@@ -44,6 +44,9 @@ int main(int, char *[])
     // Test basic functionnalities
     checkClassWithClassName<TestObject>();
 
+    // Force the use of the approximation to avoid timeout
+    ResourceMap::SetAsBool("MaximumEntropyOrderStatisticsDistribution-UseApproximation", true);
```
Comment (jschueller) [inline lib/test/t_MaximumEntropyOrderStatisticsCopula_std.cxx]: perhaps we could add a line in the docstring mentioning that key

**Comment by jschueller** [comment]
todo: backport commits on MaximumEntropyOrderStatisticsDistribution & OrthogonalUniVariatePolynomial

**Comment by jschueller** [comment]
can you rebase ?

**Comment by regislebrun** [comment]
done

File: lib/src/Base/Graph/DrawableImplementation.cxx [hunk 3]
```diff
@@ -1430,6 +1430,7 @@ Scalar DrawableImplementation::getTextSize() const
 
 void DrawableImplementation::setTextSize(const Scalar )
 {
+  (void) size;
```
Comment (jschueller) [inline lib/src/Base/Graph/DrawableImplementation.cxx]: this could fail as size is undefined

**Comment by regislebrun** [comment]
@jschueller Is it you who commented the `offset` argument in many places? IMO these arguments should be used in the code of `__str__()` and not just ignored! It breaks my fixes in many places, not cool...

**Comment by jschueller** [comment]
some tests fail now

**Pull request merged**